### PR TITLE
Added Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
🙂

### CODE REVIEW

File: po/LINGUAS [hunk 1]
```diff
@@ -1 +1,2 @@
-de
\ No newline at end of file
+de
+nl
```
Comment (sp1ritCS) [inline po/LINGUAS]: I guess a newline counts as whitespace, but I'd rather have these space seperated

File: po/nl.po [hunk 2]
```diff
@@ -0,0 +1,255 @@
+# FIDEI GETTEXT TRANSLATION TEMPLATE.
+# Copyright (C) 2022 Florian "sp1rit" <sp1rit@national.shitposting.agency>
+# This file is distributed under the same license as Fidei.
+# Florian "sp1rit" <sp1rit@national.shitposting.agency>, 2022.
```
Comment (sp1ritCS) [inline po/nl.po]: mind dropping/replacing the the first line as this is not a template anymore and update the copyright + author to 2024 and yourself?

**Review by sp1ritCS — CHANGES_REQUESTED** [review]
What is missing is a regex matching the text that ought to be small caps in https://github.com/sp1ritCS/fidei/blob/a78f14e4146595d270a9527b9edca1b2f90dcc56/data/arpa.sp1rit.Fidei.gschema.xml#L46

File: po/LINGUAS (hunk 1 above)
Comment (Vistaus) [reply po/LINGUAS]: Ah, normally devs want a new line there, but if you'd rather have a whitespace, I'll arrange that. 🙂

File: po/nl.po (hunk 2 above)
Comment (Vistaus) [reply po/nl.po]: Oh, seems like Poedit didn't replace that for some reason. :/ Will do that.

Comment (sp1ritCS) [reply po/nl.po]: please update the copyright year and change the author to yourself.

also see the small-caps regex https://github.com/sp1ritCS/fidei/pull/2#pullrequestreview-2425052698

Comment (Vistaus) [reply po/nl.po]: Hmm, you point out stuff that no one ever points out. Good catch! 🙂

What do you mean with the small-caps regex?

Comment (sp1ritCS) [reply po/nl.po]: In classical typesetting, "Lord" is generally set in small caps (see [this](https://www.gotquestions.org/LORD-GOD-Lord-God.html)).

While Unicode technically has small caps codepoints, they generally aren't used and replaced with an all caps "LORD".

Fidei matches those all-caps occurrences based on a regex for each language and converts it back into small caps, as it looks better.

While users can add languages / change the expressions themselves in the preferences, I'd like to have the defaults reflect the supported languages.

Comment (Vistaus) [reply po/nl.po]: But I don't really understand what I need to change.

Comment (sp1ritCS) [reply po/nl.po]: `data/arpa.sp1rit.Fidei.gschema.xml` has the linked `small-caps` key, whose format maps a two letter language code onto a regular expression. The default currently contains expression for `en`, `de` and `la` and you'd add yours to that default map.

If you have issues with writing/validating regular expressions, you can try it first on https://regex101.com

Comment (Vistaus) [reply po/nl.po]: Thanks for the instructions. I've done that now.

**Comment by sp1ritCS** [comment]
Thanks

**Comment by Vistaus** [comment]
Thank you for merging! 🙂

**Pull request merged**